src/timely-util/src/buffer.rs

```diff
@@ -73,6 +73,18 @@ where
         }
     }
 
+    #[inline]
+    /// Provides an iterator of elements to the buffer
+    pub fn give_iterator<I: Iterator<Item = (D, T, R)>>(
+        &mut self,
+        cap: &CapabilityRef<T>,
```

Comment:
You might have to rebase to avoid merge skew. I think this type was removed from Timely and is called `InputCapability` now.

Reply:
Thanks for the catch! I have changed this reference to `InputCapability` now as well, similarly to `give`.